aircan/dependencies/google_cloud/bigquery_handler.py

```diff
@@ -30,7 +30,8 @@ def bq_import_csv(table_id, gcs_path, table_schema):
     
 def bq_schema_from_table_schema(table_schema):
     mapping = {
-        'number': 'float'
+        'number': 'float',
+        'year': 'integer'
```

Comment:
what about all the other cases in frictionless like `yearmonth`? Have you rigorously checked the list of types and mapped all of them?

Reply:
I will do the full types mapping by the docs here (https://specs.frictionlessdata.io/table-schema/#types-and-formats)